domains/authorization/Authorization.API/API.UnitTests/API.UnitTests/[Commands]/CreateClientCommandTest.cs

```diff
@@ -0,0 +1,31 @@
+using API.Authorization._Features_;
+using API.Models;
+using API.UnitTests.Repository;
+using API.ValueObjects;
+using FluentAssertions;
+
+namespace API.UnitTests._Commands_;
+
+public class CreateClientCommandTest
+{
+    private readonly FakeClientRepository _fakeClientRepository = new();
+    private readonly FakeUnitOfWork _fakeUnitOfWork = new();
+
+    [Fact]
+    public void GivenValidInput_WhenCreatingClient_ThenClientIsCreated()
```

Comment:
What about invalid input?

Reply:
Can really see what invalid input to throw at it.
new IdpClientId(Guid.NewGuid()), new ClientName("Test Client"), ClientType.Internal, "http://localhost:5000"

URL is not always needed since internal clients dont need it. 3 other fields have strong types.